integrations/clickup/main.py

```diff
@@ -0,0 +1,88 @@
+from enum import StrEnum
+from loguru import logger
+from typing import Any
+
+from clickup.client import ClickupClient
+from port_ocean.context.ocean import ocean
+from port_ocean.core.ocean_types import ASYNC_GENERATOR_RESYNC_TYPE
+
+
+class ObjectKind(StrEnum):
+    TEAM = "team"
+    PROJECT = "project"
+    ISSUE = "issue"
+
+
+def create_client() -> ClickupClient:
+    return ClickupClient(ocean.integration_config["clickup_personal_token"])
+
+
+async def setup_application() -> None:
+    config = ocean.integration_config
+    app_host = config.get("app_host")
+    if not app_host:
+        logger.warning(
+            "No app host provided, skipping webhook creation. "
+            "Without setting up the webhook, the integration will not export live changes from Clickup"
+        )
+        return
+
+    client = create_client()
+    await client.create_webhook_events(app_host)
+
+
+@ocean.on_resync(ObjectKind.TEAM)
+async def on_resync_teams(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
```

Comment:
the return type hint has to be `RAW_RESULT` since the get teams is now returning a list of objects. Please apply it to the other resync events except the issues kind

https://github.com/port-labs/ocean/blob/main/port_ocean/core/ocean_types.py